include/CppUTest/UtestMacros.h

```diff
@@ -200,6 +200,29 @@
 #define UNSIGNED_LONGS_EQUAL_LOCATION(expected, actual, text, file, line)\
   { UtestShell::getCurrent()->assertUnsignedLongsEqual((unsigned long)expected, (unsigned long)actual, text, file, line); }
 
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
Not really, because the question is which situation happens most often and what do we ask the user.

So, two scenarios:

1) People call LONG_LONG when their system doesn't support that. How likely? Can't imagine anyone doing that, except when you port code. But then the long long declaration should give an error already

2) People call LONG_LONG without defining USE_LONG_LONG. How likely? Extremely likely, will happen for every installation on Mac and Linux. Can be fixed by letting the user manually define LONG_LONG. Do we want to do that. Not really.

Then, simply related to code clarity, NOT having a #idef is always preferred.

Now, unless you can convince me that scenario #1 is very common too... I'd suggest to remove it :)